services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -36,18 +36,7 @@ export const ExportedModalDrawerReportSection = ({
   return (
     <Box mt="2rem" data-testid="exportedModalDrawerReportSection">
       {sectionHeading && (
-        <Heading as="h3" sx={sx.childHeading}>
-          {sectionHeading}
-        </Heading>
-      )}
-      {existingEntity && verbiage?.intro?.info && (
-        <Box sx={sx.intro}>{parseCustomHtml(verbiage.intro.info)}</Box>
-      )}
-
-      {existingEntity && verbiage?.intro?.spreadsheet && (
-        <Box sx={sx.spreadSheet}>
-          <SpreadsheetWidget description={verbiage.intro.spreadsheet} />
-        </Box>
```

Comment:
Hmm... need to think about how to handle this. For modals, if it doesn't have any entities currently, it needs to only show the heading. Plenty of options -- I'm sure whatever you come up with will be great.

Reply:
Could even just add another optional prop that's like: `shouldShowExtras` or something (but maybe with a better name). Prop could default to `= true` when initially passed into the args so standard and drawer don't have to pass anything in, but modal could pass in like `entityCount` so if it's 0, they don't get rendered and if it's greater than 0 they do. 🤔

Reply:
I went with no types for the parameters, but I'm open to suggestions (this is more because I'm not sure how to best do this one). For the modals, there is already a boolean that indicates whether we have entities so I'm just reusing that!